config/webpack-config.js

```diff
@@ -16,33 +16,35 @@ const UglifyWebpackPlugin = require( 'uglifyjs-webpack-plugin' );
 const JS_ROUTES_PATH = '/js/routes';
 const COMMON_BUNDLE_NAME = 'common.js';
 
+// Commmon webpack 'module' option used in each configuration.
+// Runs code through Babel and uses global supported browser list.
+const COMMON_MODULE_CONFIG = {
+  loaders: [ {
+    test: /\.js$/,
+    loaders: [ {
+      loader: 'babel-loader?cacheDirectory=true',
+      options: {
+        presets: [ [ 'env', {
+          targets: {
+            browsers: environment.getSupportedBrowserList( 'js' )
+          },
+          debug: true
+        } ] ]
+      }
+    } ],
+    exclude: {
+      test: /node_modules/,
+      // The below regex will capture all node modules that start with `cf`.
+      exclude: /node_modules\/cf(.+)/
+    }
+  } ]
+};
+
 const modernConf = {
   cache: true,
   context: path.join( __dirname, '/../', paths.unprocessed, JS_ROUTES_PATH ),
   entry: scriptsManifest.getDirectoryMap( paths.unprocessed + JS_ROUTES_PATH ),
-  module: {
-    rules: [ {
-      test: /\.js$/,
-      use: [ {
-        loader: 'babel-loader?cacheDirectory=true',
-        options: {
-          presets: [ [ 'env', {
-            targets: {
-              browsers: environment.getSupportedBrowserList( 'js' )
-            },
-            debug: true
-          } ] ]
-        }
-      } ],
-      exclude: {
-        test: /node_modules/,
-        // TODO: Move this into a config variable so that we can easily add
```

Comment:
@contolini I _think_ I addressed your TODO here by moving this into a constant shared by all configs. Is that right?

Reply:
Yeah works for me. 👍